databricks/koalas/generic.py

```diff
@@ -297,12 +297,12 @@ def cumprod(self, skipna: bool = True):
         """
         return self._apply_series_op(lambda kser: kser._cumprod(skipna))  # type: ignore
 
+    # TODO: Since this method has removed pandas >= 1.0.0,
+    # so we should be block this from using it directly via external API
     def get_dtype_counts(self):
         """
```

Comment:
Although `get_dtype_counts` has removed since `pandas>=1.0.0`, we still need this for `DataFrame.info`.

Reply:
we decided keep this at `Deprecated functions`